### PR TITLE
Add Profile screen

### PR DESCRIPTION
**Pull request by Juett**
add Profile Screen with setAReminder Component and link to EditProfile

### CODE REVIEW

File: js/App.js [hunk 1]
```diff
@@ -2,6 +2,26 @@ import React, { Component } from "react";
 import AppNavigator from "./navigation/RootStackNavigator";
 
 export default class App extends Component {
+  // callingAPI = async () => {
```
Comment (10thfloor) [inline js/App.js]: Commented code should not be submitted in PRs. Thanks!

File: js/components/SetAReminder/SetAReminder.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    };
+  }
+  setDate = newDate => {
+    console.log(newDate)
+    this.setState({ chosenDate: newDate });
+  };
+  toggleReminderSwitch = value => {
+    this.setState({ reminderSwitch: value });
+  };
+  setModalVisible(visible) {
```
Comment (10thfloor) [inline js/components/SetAReminder/SetAReminder.js]: Stick to one style of class method. If you're using static methods with arrows, make sure all your methods are as such.

File: js/components/SetAReminder/SetAReminder.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+          animationType="fade"
+          transparent={true}
+          visible={this.state.modalVisible}
+        >
+          <View style={styles.modalContentContainer}>
+            <LinearGradient
+              useAngle={true}
+              angle={180}
+              angleCenter={{ x: 0.5, y: 0.5 }}
+              colors={["#F3E3D8", "#fff"]}
```
Comment (10thfloor) [inline js/components/SetAReminder/SetAReminder.js:49]: Linear gradient colours should be abstracted to global styles. Talk with @ajaguasin to get this sorted.

File: js/components/SetAReminder/styles.js [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    fontSize: globalStyles.textFont.fontSize,
+    fontWeight: globalStyles.textFont.fontWeight,
+    flexDirection: "row",
+    justifyContent: "space-between",
+    alignItems: "center",
+    width: "100%",
+    marginBottom: 5
+  },
+  reminderRow2: {
+    // fontFamily: globalStyles.textFont.fontFamily,
```
Comment (10thfloor) [inline js/components/SetAReminder/styles.js:63]: Remove the commented out styles form this file, please.

File: js/config/styles.js [hunk 5]
```diff
@@ -1,25 +1,30 @@
...
-    fontSize: 13
+    fontSize: 13,
+    fontWeight: "900"
   },
 
-  greyTextColor: { color: "#5D5D5D" },
-  blueGradientColor: { start: { color: "#A4E8FF" }, end: { color: "#6BCCFD" } },
-  pinkGradientColor: { start: { color: "#FFB5BA" }, end: { color: "#FBE3D7" } }
+  greyTextColor: { color: "#5D5D5D" }
+  // blueGradientColor: { start: { color: "#A4E8FF" }, end: { color: "#6BCCFD" } },
```
Comment (10thfloor) [inline js/config/styles.js]: Work with @ajaguasin to consolidate global styles. (And, remove comments).

File: js/lib/API/index.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+  const defaultHeaderOptions = {
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    },
+  };
+
+  const headerOptions = {
+    method: 'GET', //defalut method
```
Comment (10thfloor) [inline js/lib/API/index.js]: defalut ?

File: js/screens/EditProfile/EditProfile.js [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+    <ImageBackground
+      source={require("../../assets/Background/background.png")}
+      style={{ width: "100%", height: "100%" }}
+    >
+      <ScrollView>
+        <View style={styles.editProfileContainer}>
+          <Image
+            style={{ width: 120, height: 120 }}
+            source={{ uri: "https://speakbox.ca/images/logo.png" }}
+            // source={require("../../assets/Avatars/beachBear.svg")}
```
Comment (10thfloor) [inline js/screens/EditProfile/EditProfile.js:17]: Remove comments.

File: js/screens/Profile/Profile.js [hunk 8]
```diff
@@ -1,12 +1,77 @@
...
+    <ImageBackground          
+      source={require("../../assets/Background/background.png")}
+      style={{ width: "100%", height: "100%" }}
+    >
+      <ScrollView>
+        <View style={styles.profileContainer}>
+          <Image
+            style={{ width: 120, height: 120 }}
+            source={{ uri: "https://speakbox.ca/images/logo.png" }}
+            // source={require("../../assets/Avatars/beachBear.svg")}
```
Comment (10thfloor) [inline js/screens/Profile/Profile.js:26]: Remove comments.

File: js/screens/Profile/Profile.js [hunk 9]
```diff
@@ -1,12 +1,77 @@
...
+            onPress={() => {
+              navigation.navigate("EditProfile");
+            }}
+          >
+            <Text style={styles.editAccount}>Edit Account</Text>
+          </TouchableOpacity>
+          <View style={styles.plantContainer}>
+            <Text style={styles.plantName}>[Plant Name] Progress</Text>
+            {/* BABY FLOWER
+        <Image
```
Comment (10thfloor) [inline js/screens/Profile/Profile.js:41]: Remove comments!

File: js/screens/Profile/styles.js [hunk 10]
```diff
@@ -1,2 +1,74 @@
...
+
+const styles = StyleSheet.create({
+  profileContainer: {
+    height,
+    width,
+    alignItems: "center",
+    justifyContent: "space-around"
+  },
+  accountName: {
+    // fontFamily: globalStyles.titleFont.fontFamily,
```
Comment (10thfloor) [inline js/screens/Profile/styles.js:13]: Remove comments.

**Review by 10thfloor — CHANGES_REQUESTED** [review]

File: js/components/SetAReminder/SetAReminder.js (hunk 3 above)
Comment (Juett) [reply js/components/SetAReminder/SetAReminder.js:49]: this is the only time this gradient will be used.

File: js/components/SetAReminder/styles.js (hunk 4 above)
Comment (Juett) [reply js/components/SetAReminder/styles.js:63]: the font families have not been added to the project yet. as soon as they are these lines will be uncommented so to apply the correct font.

File: js/screens/EditProfile/EditProfile.js (hunk 7 above)
Comment (Juett) [reply js/screens/EditProfile/EditProfile.js:17]: the comment is the correct image but it is an svg and we have yet found a way to make it work.

File: js/screens/Profile/Profile.js (hunk 8 above)
Comment (Juett) [reply js/screens/Profile/Profile.js:26]: the comment is the correct image but it is an svg and we have yet found a way to make it work.

File: js/screens/Profile/Profile.js (hunk 9 above)
Comment (Juett) [reply js/screens/Profile/Profile.js:41]: the images are all the different stages of the plant. Once we have realm set up I will need to render a different plant based on the user's streak.

File: js/screens/Profile/styles.js (hunk 10 above)
Comment (Juett) [reply js/screens/Profile/styles.js:13]: the font families have not been added to the project yet. as soon as they are these lines will be uncommented so to apply the correct font.

**Review by 10thfloor — DISMISSED** [review]

**Pull request merged**